kikit/fab/pcbway.py

```diff
@@ -147,9 +170,11 @@ def exportPcbway(board, outputdir, assembly, schematic, ignore,
     typeFields          = [x.strip() for x in soldertype.split(",")]
     footprintFields     = [x.strip() for x in footprint.split(",")]
     addVirtualToRefsToIgnore(refsToIgnore, loadedBoard)
+    posData = collectPosData(loadedBoard, correctionFields, bom=components)
     bom = collectBom(components, manufacturerFields, partNumberFields,
                      descriptionFields, notesFields, typeFields,
-                     footprintFields, refsToIgnore)
+                     footprintFields, refsToIgnore, posData)
+    posData = ignoreUnusedPosition(posData, bom)
```

Comment:
Here you can use https://github.com/yaqwsx/KiKit/blob/2e56d48dc8de2ea5e1867a25d78beab063e3ec3d/kikit/fab/jlcpcb.py#L77-L78